responsive slider

```diff
@@ -0,0 +1,160 @@
+<DOCTYPE html>
```

Comment:
what is the extension of this file? it just says "responsive slider".

Reply:
created responsiveslider.html is that what you meant by file extension? Will add Tuki as a reviewer.

Reply:
hi @RideOrDye, yes, that's correct. I just checked out your repo and pulled the slider-js-component-part-2 branch to my local to take a look. 
I see that you created a file called `"responsiveslider.html"` however you still have another file called `"responsive slider"` as well in your repo. That additional file can be deleted now.
I also have a suggestion to move your css and js code into their own files and attach them to your html file.
(It is not a mistake to keep them in the html file, it is just a better practice to separate them.) 
Also, please rename your repo from `"MainClearMindRepo"` to `"clear-mind-react"` as it was one of the first week's requirements.

I also took a closer look at your code. I really love that you made the slider numbers respond to the color update, it is a very nice design element. 
I suggest to investigate the functionality that is supposed to resize the slider component. Right now that part seems to be incomplete. 
Great work otherwise and nice design🤩